Plutarch/Builtin/ByteString.hs

```diff
@@ -123,15 +189,17 @@ consisting entirely of zero bytes.
 @since WIP
 -}
 pzeroesBS :: forall (s :: S). Term s (PInteger :--> PByteString)
-pzeroesBS = punsafeBuiltin PLC.ReplicateByte
+pzeroesBS = phoistAcyclic $ plam $ \len ->
+  preplicateBS # len #$ pintegerToByte # pconstantInteger 0
```

Comment:
We don't want to have these here. Plutarch.Builtin.* must stay without any "additional checks"

Reply:
If we want to have "safe" version, that should go somewhere else other than builtin modules. We only keep "raw" versions here

Reply:
We can probably just remove these. They don't do anything you can't do with builtins anyway.